sandbox-tools/merge-repos/src/config.ts

```diff
@@ -194,7 +194,8 @@ export let addMissingDevDeps = {
  * Remove these dependencies (required or dev)
  */
 export let dropDependencies = {
-    "lerna": ""
+    "lerna": true,
+    "rimraf": true
```

Comment:
By having `rimraf` included in the package.json when rush attempts to create the symlinks for a project there are multiple versions which causes `rush` to halt rather than just overwriting the previous version.